quickstarts/EAP7-1941-add-jaxrs-jwt-to-EAP-quickstarts.adoc

```diff
@@ -0,0 +1,117 @@
+= Add jaxrs-jwt to the EAP quickstarts
+:author:            Farah Juma
+:email:             fjuma@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The jaxrs-jwt quickstart is currently a community-only quickstart because it's not part of the EAP quickstarts repo.
+Nothing else demonstrates how to configure this so it should be added to the EAP quickstarts as well.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/EAP7-1940[EAP7-1940]
+* https://issues.redhat.com/browse/WFLY-16472[WFLY-16472]
+
+=== Related Issues
+
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
```

Comment:
Does are these mode corrects? 

- I don't see a Managed domain and bootable-jar instrusction in the WildFly QS. I think this isn't supported case.
- OpenShift Instruction are there but the helm chart is missing in the WildFly QS.

Reply:
Fixed.

Reply:
@fjuma can you please revert the removal of OpenShift support, the support is there already, this was simply a bug that I am fixing now at [WFLY-19719](https://issues.redhat.com/browse/WFLY-19719)

Reply:
Fixed.